e2e-tests/tests/features/dbbackup_psmdb/20-assert.yaml

```diff
@@ -10,14 +10,6 @@ spec:
   backupStorageName: test-storage
   dbClusterName: test-psmdb-cluster
 ---
-apiVersion: everest.percona.com/v1alpha1
-kind: DatabaseClusterBackup
-metadata:
-  name: test-db-backup-azure
-spec:
-  backupStorageName: test-storage-azure
-  dbClusterName: test-psmdb-cluster
----
```

Comment:
two storages can not exist simultaneously from now on, so I moved the azure-related stuff to a separate step

Reply:
What is the reason we don't support multiple backup storage locations?

Reply:
because PITR can not be enabled for PSMDB when multiple storages are used.

Reply:
it's a PBM limitation

Reply:
Does this change prevent users from having multiple backup locations with different policies set per S3 bucket, such as expiration time?
If yes, I think we shall rethink the approach.

Reply:
I believe the different policies mean the different buckets, which in terms of Everest means the different `BackupStorages`, so yes, it's gonna prevent users for having that. I understand the concern. 

It's again more like a product question - what is more important, the PITR or the multi storages support. Waiting for the product decision.

Reply:
UPD: 
The [discussion](https://perconacorp.slack.com/archives/C0545J2BEJX/p1705491681958579) with Product folks and the Cloud team. 
**Conclusion:** move forward with the one storage limitation, created [the ticket](https://perconadev.atlassian.net/browse/PBM-1228) for PBM team to support multiple storages.